dozer/cogs/info.py

```diff
@@ -66,11 +105,16 @@ async def guild(self, ctx):
         e.add_field(name='Created at', value=guild.created_at.strftime(datetime_format))
         e.add_field(name='Owner', value=guild.owner)
         e.add_field(name='Members', value=guild.member_count)
-        e.add_field(name='Channels', value=len(guild.channels))
-        e.add_field(name='Roles', value=len(guild.roles) - 1)  # Remove @everyone
-        e.add_field(name='Emoji', value=len(guild.emojis))
+        e.add_field(name='Channels', value=str(len(guild.channels)))
+        e.add_field(name='Roles', value=str(len(guild.roles) - 1))  # Remove @everyone
+        e.add_field(name='Emoji', value=str(len(guild.emojis)))
```

Comment:
It would be useful to count static vs animated emojis and show those totals here as well.

Reply:
Maybe, I'll go steal more code